### DISCUSSION TITLE
Large Molecule PK curve shape

### DISCUSSION THREAD

**Discussion by akim2000** [Open-Systems-Pharmacology/Forum#1707] [Questions & Problems]
I'm trying to make a brain model for large molecules (mAb), but fitting the plasma curve is a bit hard. 

Currently, I have some observed data of a molecule and I am trying to adjust the Kd (FcRn) in endosomal space of the molecule to fit the plasma curve. But as seen in the picture below, I am not sure how to get the curve shape (red line) to be steeper at the beginning then plateau. 

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/139281283/8d6d80c7-dd25-4024-ba10-68396bef8d6c)

From what I know, metabolism of large molecules (mAbs) in PK-Sim is mainly dictated by the endosomal clearance, and FcRn recycling also plays a role in this. The rate of specific clearance is determined by (rate of uptake - rate of recycling). I can lower the rate of recycling to steepen the curve and fit the observed data better, but that wouldn't really make sense physiologically to have different rates of recycling for different molecules, or would that be okay? Are there any other mechanism/parameters that I am missing here regarding large molecules? 

Below is an example of fitting the plasma curve (Red line at top) by changing Kd and rate of recycling at quite extreme values (Kd = 0.02µmol/L, recycling = 0.0002477 1/min)

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/139281283/79885fd7-4fd6-4db9-a8a3-d2a5ebf3e723)

This works well for this specific molecule, but for another, the curve just does not fit when keeping the rate of recycling constant. Are there any other parameters that would make more sense to change per molecule other than Kd? Ultimately, I am trying to see if I can predict CSF conc. by using plasma conc. data and fitting the curve to just the plasma conc. via Kd, but maybe I need to fit more than just the Kd parameter.

**Comment by Christoph27** [comment]
What about target binding of the antibody? This might explain the shape...

**Reply by akim2000** [reply]
These do have different targets and I'm currently thinking about the best way to implement target binding.

**Comment by prvmalik** [comment]
What species is this? Model is challenging to use in Rats at this time.

Curve shape looks unusual to me - are you using the recommended 5.34 nm hydrodynamic radius for a mAb? Is the FcRn binding affinity (Kd) in the model within a standard range (i.e., somewhere between 100 nM and 1.5 uM for most mAbs?). 

Paul

**Reply by akim2000** [reply]
It is rats. The curve shape is interesting as I have messed around with different parameters to see what values can even fit such observed data.
